ts/features/walletV3/common/utils/index.ts

```diff
@@ -100,3 +104,20 @@ export const isPaymentSupported = (
     O.getOrElseW(() => "notAvailable" as const)
   );
 };
+
+export const getPaymentLogo = (
+  selectedMethod: WalletInfoDetails
+): IOLogoPaymentType => {
+  switch (selectedMethod.type) {
+    case TypeEnum.CARDS:
+      const cardsType = selectedMethod as WalletInfoDetails1;
+      return cardsType.brand.toLowerCase() as IOLogoPaymentType;
+    case TypeEnum.PAYPAL:
+      return "payPal";
+    default:
+      return "visa";
```

Comment:
I think this util could benefit from the usage of the `findFirstCaseInsensitive` util, also I don't really know how good it is to return `visa` as default; can we make it return undefined? sadly when mapping just to `IOLogoPayment` we can't really make it return a generic

Reply:
It would be awesome if the `ModuleCheckout` had the "generic" card logo to show if the card type is not the one listed, but I think it could be improved in the next iteration, in the meanwhile it has been addressed into 619b12275c3c1f13ef2be1d69d31d41e41d98283 👍